### DISCUSSION TITLE
Adding blacknull to path

### DISCUSSION THREAD

**Discussion by turtlejuno** [shekohex/coemu#90] [Q&A] [answered]
Reading the ReadMe, there's a step to add "blacknull" at the end of the path for the shortcut. I'm not understanding the step. Could someone elaborate on how exactly to do this?

**Comment by shekohex** [comment, marked as answer]
You can follow the steps mentioned here: https://web.archive.org/web/20240407162939/https://www.digitalcitizen.life/shortcut-arguments-parameters-windows/

Another way, is to use Conquer Loader V5 from here: https://www.elitepvpers.com/forum/co2-programming/610999-release-conquerloader-v2.html which is the easiest way to get started.

> You may want to add it to your Windows definder allow list / execlude it from scanning, because it sees it as a vairus/injector.
